pkg/sql/pg_catalog.go

```diff
@@ -264,11 +265,12 @@ CREATE TABLE pg_catalog.pg_class (
 	populate: func(ctx context.Context, p *planner, prefix string, addRow func(...tree.Datum) error) error {
 		h := makeOidHasher()
 		return forEachTableDesc(ctx, p, prefix, func(db *sqlbase.DatabaseDescriptor, table *sqlbase.TableDescriptor) error {
-			// Table.
+			// The only difference between tables, views and sequences is the relkind column.
 			relKind := relKindTable
 			if table.IsView() {
-				// The only difference between tables and views is the relkind column.
 				relKind = relKindView
+			} else if table.IsSequence() {
+				relKind = relKindSequence
```

Comment:
is there any way we could error if none of the existing relKinds match the given object? or is that impossible given the info we have access to? this just seems like something that would be easy to forget to update

Reply:
Yeah I was thinking about this. Probably the thing to do is to have an enum on the table descriptor saying what kind of object it is (view, table, or sequence), then we could replace the if/else trees that have arisen in a few places in the code with switch statements.

That would help here, because we could have a map from the enum values to relkind values, and do this as a lookup in that map. Then, there's probably a way to write a unit test which fails if the map has fewer entries than the enum. (-1, since indices aren't a type of table descriptor but do show up in pg_class).

Anyway, gonna save that enum work for a later PR.